Xamarin.Forms.Platform.iOS/CollectionView/UICollectionViewDelegator.cs

```diff
@@ -19,6 +22,45 @@ public UICollectionViewDelegator(ItemsViewLayout itemsViewLayout, ItemsViewContr
 			ItemsViewController = itemsViewController;
 		}
 
+		public override void DraggingStarted(UIScrollView scrollView)
+		{
+			_horizontalOffset = (float)scrollView.ContentOffset.X;
+			_verticalOffset = (float)scrollView.ContentOffset.Y;
+		}
+
+		public override void DraggingEnded(UIScrollView scrollView, bool willDecelerate)
+		{
+			_horizontalOffset = 0;
+			_verticalOffset = 0;
+		}
+
+		public override void Scrolled(UIScrollView scrollView)
+		{
+			var indexPathsForVisibleItems = ItemsViewController.CollectionView.IndexPathsForVisibleItems.OrderBy(x => x.Row);
```

Comment:
Index paths do not arrive sorted, so we sort them here before reporting them to `Core`.